pkg/core/region.go

```diff
@@ -1789,6 +1789,29 @@ func (r *RegionsInfo) GetRegionCount(startKey, endKey []byte) int {
 	defer r.t.RUnlock()
 	start := &regionItem{&RegionInfo{meta: &metapb.Region{StartKey: startKey}}}
 	end := &regionItem{&RegionInfo{meta: &metapb.Region{StartKey: endKey}}}
+
+	// check if both keys are in the uncovered ranges.
+	startItemt, sit := r.tree.tree.GetWithIndex(start)
```

Comment:
Can we avoid invoking `GetWithIndex(start)` twice since line 1816 has already been called?